dozer-integrations/dozer-spring-support/dozer-spring4/src/test/resources/spring-bean-factory.xml

```diff
@@ -1,39 +1,39 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
same as other, if nothing changed, please revert

Reply:
Line ending was changed to `LF` automatically. Git uses `LF`.